lib/java/desktop-environment/src/main/java/org/enso/desktopenvironment/WindowsTrashBin.java

```diff
@@ -0,0 +1,97 @@
+package org.enso.desktopenvironment;
+
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.List;
+import org.graalvm.nativeimage.StackValue;
+import org.graalvm.nativeimage.c.CContext;
+import org.graalvm.nativeimage.c.constant.CConstant;
+import org.graalvm.nativeimage.c.function.CFunction;
+import org.graalvm.nativeimage.c.struct.CField;
+import org.graalvm.nativeimage.c.struct.CStruct;
+import org.graalvm.nativeimage.c.type.CCharPointer;
+import org.graalvm.nativeimage.c.type.CTypeConversion;
+import org.graalvm.word.PointerBase;
+import org.slf4j.LoggerFactory;
+
+@CContext(WindowsTrashBin.ShellApi.class)
+final class WindowsTrashBin implements TrashBin {
+  @CConstant
+  public static native int FO_DELETE();
+
+  @CConstant
+  public static native int FOF_ALLOWUNDO();
+
+  @CConstant
+  public static native int FOF_NO_UI();
+
+  @Override
+  public boolean isSupported() {
+    return true;
+  }
+
+  @Override
+  public boolean moveToTrash(Path path) {
+    if (Platform.getOperatingSystem().isWindows())
+      try {
+        return moveToTrashImpl(path);
+      } catch (NullPointerException | LinkageError err) {
+        if (!Boolean.getBoolean("com.oracle.graalvm.isaot")) {
+          LoggerFactory.getLogger(MacTrashBin.class)
+              .warn(
+                  "Moving to Windows' Trash Bin is not supported in non-AOT mode. Deleting"
+                      + " permanently");
+          return path.toFile().delete();
```

Comment:
Just return false, the project manager logic will delete the project directory permanently as a fallback action.